doc/release-notes/0.17/0.17.md

```diff
@@ -0,0 +1,150 @@
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1"><thead align="left">
+<tr valign="bottom">
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Version / Platform</th>
+<th valign="bottom">Impact</th>
+</tr>
+</thead>
+<tbody>
+
+
+<tr><td valign="top"><a href="https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/324">#324</a> <a href="https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/191">#191</a></td>
+<td valign="top">OpenSSL Digest algorithm is reenabled</td>
+<td valign="top">OpenJDK8 and 11</td>
+<td valign="top">Following the resolution of issue [#5611](https://github.com/eclipse/openj9/issues/5611), the Digest algorithm is reenabled. This algorithm is already supported and enabled on OpenJDK 13.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/7001">#7001</a></td>
+<td valign="top">New shared classes cache <tt>-Xshareclasses:noPersistentDiskSpaceCheck</tt> option </td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">For file systems that do not support the checking of free space, this option causes the VM to skip the disk space check operation that is done before creating a persistent shared classes cache.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/7149">#7149</a></td>
+<td valign="top">New option <tt>-XX:[+|-]ShareUnsafeClasses</tt> </td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">Provides the ability to enable and disable the storing and finding of Unsafe classes in the shared classes cache, which are created with <tt>Unsafe.defineClass</tt>. This option is enabled by default.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/7089">#7089</a></td>
+<td valign="top">New option <tt>-XX:[+|-]ClassRelationshipVerifier</tt> </td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">Provides the ability to enable and disable the recording and validating of class relationships in the verifier. When enabled, unnecessary class loading is avoided, which can reduce VM startup time. This option is disabled by default and cannot be used with <tt>-Xfuture</tt>, which is also enabled when the <tt>-Xverify:all</tt> option is set. </td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/7213">#7213</a></td>
+<td valign="top">New experimental <tt>-Xshareclasses</tt> sub-options for Docker containers</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">The <tt>-Xshareclasses:createLayer</tt> and <tt>-Xshareclasses:layer=<number></tt> options are intended
+for users who are running Java applications in docker containers. These options allow the shared classes cache to be
+built into different layers of a docker image. These options are experimental.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/7213">#7213</a></td>
```

Comment:
This looks like the wrong issue number. I think it should be #5884.

Reply:
Sorry - copied from the line above. Now changed.